### DISCUSSION TITLE
Wrong rank shown on own panel in multi (not playlists)

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16059] [Q&A] [answered]
Well, now the rank is there but it's not correct, happened every time i think but i didn't take a screenshot for all of them. 
I've only played 1 playlist but it's correct there so i didn't play more of those 

![Screenshot_32](https://user-images.githubusercontent.com/53081750/145727886-36e7922c-b238-4599-829a-f6fa876cedc6.png)
![Screenshot_33](https://user-images.githubusercontent.com/53081750/145727887-bc902d75-8af3-4d68-ae94-4ee14987d97d.png)
![Screenshot_31](https://user-images.githubusercontent.com/53081750/145728001-bda638f8-301e-425b-a5e4-efb2bbaefb74.png)

I also had a score failing to submit and there it even didn't show any rank. (logs and screenshot are from different runs but same room)
At 2021-12-12 20:05:22 [error]: Failed to submit score
[network.log](https://github.com/ppy/osu/files/7699687/network.log)
[database.log](https://github.com/ppy/osu/files/7699686/database.log)
[performance.log](https://github.com/ppy/osu/files/7699684/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7699685/runtime.log)

**Comment by smoogipoo** [comment, marked as answer]
Can you explain how the rank is incorrect?

**Reply by Theighlin** [reply]
Well, you can see that on the first 2 images it's showing # 2 on me when it should be # 4, right? and on the third image it goes 1, 2, 2 and not 1, 2, 3

**Reply by smoogipoo** [reply]
Aha, thank you for that. I was looking at the grades (A / B / C / D) and not finding anything weird.

**Reply by smoogipoo** [reply]
The incorrect ranking will be resolved in https://github.com/ppy/osu/pull/16061.
